### PR TITLE
Mcda only no moga to master

### PR DESCRIPTION
**Pull request by sythel**
v1.5.0 -> master (was v2.0.0)

### CODE REVIEW

File: src/dams_mcda/ui.R [hunk 1]
```diff
@@ -1132,12 +1088,12 @@ ui <- shinyUI(fluidPage(
...
-					remember seeing these data when you clicked on the link for the data for this dam during the preference elicitation in Step 3. We include the raw data values again here to help make the MCDA calculations \
-					more transparent, so you can clearly see what goes into the final calculation that produces Figures 1 and 2 above. In addition, you can use this table to sort decision alternatives in ascending or descending order\
-					in each column by clicking on the arrow next to the column header . Note: fish survival values shown here are discrete, but in reality, the values are network-dependent and would be impacted by upstream or downstream\
-					changes. They are presented here as the average of a range of possible values for this dam, depending on what happens at other dams. This interaction between decisions at other dams and these decision criteria are\
-					modeled in the multi-objective optimization that leads to the final Map Recommendation.<br>"
+				  "<br><b>Results Interpretation</b> for Table 2. This table displays the raw data values we collected and/or calculated/generated through our research for each decision criterion and alternative. You may \
```
Comment (elbfox) [inline src/dams_mcda/ui.R]: Need to take out Lines  1095 (starting with ""depending on what happens at other dams...") to 1096.

File: src/dams_mcda/server.R [hunk 2]
```diff
@@ -1656,7 +1569,7 @@ server <- function(input, output, session) {
 			"Total MCDA Score", # y axis label
```
Comment (elbfox) [inline src/dams_mcda/server.R]: Can you change the label on this to say "Final MCDA  Score", please?

**Comment by sythel** [comment]
> I made some small change requests but my only real comment here is that we don't need (unless you're using it somewhere...?) RMatlab with this version.

correct was only keeping around as a backup

File: src/dams_mcda/ui.R (hunk 1 above)
Comment (sythel) [reply src/dams_mcda/ui.R]: 1095 is mid sentence

**Comment by sythel** [comment]
launching on shiny2 shortly

**Pull request merged**